Lib/test/test_timeit.py

```diff
@@ -387,6 +403,44 @@ def callback(a, b):
                     '500 0.488\n')
         self.assertEqual(s.getvalue(), expected)
 
+    def test_autorange_with_callback_with_argument(self):
+        def callback(a, b):
+            print("{} {:.3f}".format(a, b))
+        with captured_stdout() as s:
+            num_loops, time_taken = self.autorange_with_argument(callback=callback,
+                    total_time=0.6)
+        self.assertEqual(num_loops, 1000)
+        self.assertEqual(time_taken, 1000/1024)
+        expected = ('1 0.001\n'
+                    '2 0.002\n'
+                    '5 0.005\n'
+                    '10 0.010\n'
+                    '20 0.020\n'
+                    '50 0.049\n'
+                    '100 0.098\n'
+                    '200 0.195\n'
+                    '500 0.488\n'
+                    '1000 0.977\n')
+        self.assertEqual(s.getvalue(), expected)
+
+    def test_autorange_with_callback_with_argument(self):
```

Comment:
Same as above regarding the name.
I'm also not sure it's worth duplicating everything just to test 2 different values (0.4 and 0.6).
If testing different values makes sense and doesn't take too much time, you could create an helper method or use a loop (possibly using `subTest`).
Perhaps a test with negative values should be added as well.